### ISSUE TITLE
Bug: The tool doesn't download all releases (record labels)

### ISSUE THREAD

**Issue by zzzzzozzzzz** [nathom/streamrip#120]
## Bug Report
**Source**
Qobuz

**Current Behavior**
The tool doesn't download all releases (record labels)

**Command(s) used:**

```bash
poetry run rip -u 'https://www.qobuz.com/gb-en/label/...'
poetry run rip -u 'https://play.qobuz.com/label/...'
```

**Environment**

- `streamrip` version: 5afcee7 | v0.7
- Python version: 3.9
- OS: Linux

**Comment by nathom** [comment]
Please attach the config file and the actual command you used.

**Comment by zzzzzozzzzz** [comment]
https://github.com/nathom/streamrip/issues/121#issuecomment-885455734

**Comment by nathom** [comment]
Do you see any output, or errors? Which albums did it not download?

**Comment by zzzzzozzzzz** [comment]
> Do you see any output, or errors? Which albums did it not download?

No output, no errors. 
The tool just downloads a certain number of releases instead of downloading them all. The reason? I don't know.


I sent you the link via ds but you can check with all links. I will never attach a Qobuz/Tidal/Deezer link on github again.

**Comment by nathom** [comment]
Does this happen on the dev branch still?

**Comment by nathom** [comment]
This should have been fixed in `v1.0`
